conf/machine/sturgeon.conf

```diff
@@ -6,10 +6,6 @@
 require conf/machine/include/tune-cortexa7.inc
 require conf/machine/include/hybris-watch.inc
 
-MACHINE_DISPLAY_ROUND = "true"
-MACHINE_HAS_WLAN = "true"
-MACHINE_HAS_SPEAKER = "true"
```

Comment:
I actually think it's good to keep those as OE variables because we might need to change the OE build based on those values (eg: if MACHINE_HAS_SPEAKER we could maybe install additional packages).
This will also make the transition of all watches to the config file easier. If someday we need to evolve the machine.conf format we don't have to change all watches.
I propose we generate a machine.conf file out of those variables in the qml-asteroid recipe instead. (I'll also leave a comment there)